proteus/mprans/RANS2P2D.h

```diff
@@ -2031,7 +2031,7 @@ namespace proteus
 						w,
 						q_velocity_sge[eN_k_nSpace+0],
 						q_velocity_sge[eN_k_nSpace+1],
-						q_velocity_sge[eN_k_nSpace+2],
+						q_velocity_sge[eN_k_nSpace+1],//cek hack, should not be used
```

Comment:
@adimako the velocity in 2D only has two components so you have to use this hack (or we could change the interface to only use u and v).